crates/cxx-qt-lib-headers/include/gui/qpainter.h

```diff
@@ -11,8 +11,13 @@
 
 #include <QtGui/QPainter>
 
+#include "rust/cxx.h"
+
+#include <cstdint>
```

Comment:
why is cstdint and rust/cxx.h required here? I thought maybe for the shared enum but `qt.h` doesn't have these and those work ?

Reply:
I think it's a copy/paste error.